app/views/volunteers/_form.html.slim

```diff
@@ -0,0 +1,115 @@
+= simple_form_for(@volunteer) do |f|
+  - if f.error_notification.present?
+    .row
+      .col-xs-12
+        = f.error_notification
+
+  .row
+    .col-xs-12.col-md-6
+        = f.input :first_name
+
+    .col-xs-12.col-md-6
+        = f.input :last_name
+
+  .row
+    .col-xs-12.col-md-6
+      = f.input :date_of_birth, as: :date, start_year: Date.today.year - 90,
+        end_year: Date.today.year, order: [:day, :month, :year], include_blank: true
```

Comment:
The order should be configured through the translations i think, but i don't know how to do this with simple_form, please make a little bit of research, i think vok could know